README.md

```diff
@@ -72,12 +65,40 @@ A valid (read: that follows this interface) IPLD format implementation the follo
 
 `callback` must have the signature `function (err, dagNode)`, where `err` is an Error if the function fails and `dagNode` is the dagNode that got deserialized in the process.
 
-#### `util.cid(dagNode, callback)`
+#### `util.cid(dagNode[, options], callback)`
 
 > get the CID of the dagNode
 
+Options include:
+  - version - the CID version to be used (defaults to 1)
+  - hashAlg - the hash algorithm to be used (default to the one set by the format)
+
 `callback` must have the signature `function (err, cid)`, where `err` is an Error is the function fails and `cid` is a CID instance of the dagNode.
 
+#### `util.cidSerialized(binaryBlob[, options], callback)`
+
+> get the CID of the serialized dagNode
+
+Options include:
+  - version - the CID version to be used (defaults to 1)
+  - hashAlg - the hash algorithm to be used (default to the one set by the format)
+
+`callback` must have the signature `function (err, cid)`, where `err` is an Error is the function fails and `cid` is a CID instance of the dagNode.
+
+#### `util.copy(dagNode, callback)`
+
+> return a copy of the dagNode
+
+`callback` must have the signature `function (err, dagNodeCopy)`, where `err` is an Error is the function fails and `dagNodeCopy` is a copy of the dagNode.
```

Comment:
Added this method. It will be useful and go-ipld has to have it anyway so better make it part of the spec.

Reply:
What is the use case? I'm hesitate to add new methods just because the seem useful.

Reply:
@vmx folks shooting themselves in the foot, mostly.